src/sentry/static/sentry/app/components/events/interfaces/frameVariables.jsx

```diff
@@ -19,16 +19,11 @@ const FrameVariables = React.createClass({
 
     for (let key in data) {
       let value = data[key];
-      children.push(<dt key={'dt-' + key}>{key}</dt>);
-      children.push((
-        <dd key={'dd-' + key}>
-          <ContextData data={value} />
-        </dd>
-      ));
+      children.push(<tr key={key}><td className="key">{key}</td><td className="value"><ContextData data={value} /></td></tr>);
```

Comment:
tbh we can probably replace this with KeyValueList too since it does the same thing